src/Xamarin.Android.Build.Tasks/Tasks/CalculateProjectDependencies.cs

```diff
@@ -53,8 +53,8 @@ public override bool RunTask ()
 				manifestApiLevel = manifest.TargetSdkVersion ?? manifest.MinSdkVersion ?? DefaultMinSDKVersion;
 			}
 			var sdkVersion = Math.Max (targetApiLevel.Value, manifestApiLevel);
-			dependencies.Add (CreateAndroidDependency ($"platforms/android-{sdkVersion}", $""));
-			dependencies.Add (CreateAndroidDependency ($"build-tools/{BuildToolsVersion}", BuildToolsVersion));
+			dependencies.Add (CreateAndroidDependency (Path.Combine ("platforms", $"android-{sdkVersion}"), $""));
```

Comment:
This must have changed behaviour in the sdk installer. I seem to remember the spec required the system to use `/` as a separator :/

Reply:
Relevant spec: https://microsoft-my.sharepoint.com/:w:/p/mhutch/ETjdNgT_yfJMszzuRRW2hUYB133Goo831A5U2rAFwgI8bA?e=77b68c4ccf5841139a242a0d99cd0368

Forward slash `/` is explicitly mentioned in the spec:

> For items in the main [Android repository manifest](https://dl-ssl.google.com/android/repository/repository2-1.xml), the ID shall be the path formatted in \*nix format, e.g. “platforms/android-25” or “ndk-bundle”. For [add-ons](http://dl-ssl.google.com/android/repository/addon.xml), the ID shall be the add-on’s “name-id” property. We will also have our own manifest format where the fileSystemPath will be exposed directly. 

We *do not* want to use `Path.Combine()` here.

Reply:
So in that case the issue must be in the task which calls the SDK Manager or the SDK Manager code itself I guess.